_test/test_config/config_base_tester.m

```diff
@@ -1,52 +1,59 @@
 classdef config_base_tester <config_base
-    %Test class representing some generig configuration
-    
+    %Test class representing some generic configuration
     properties(Dependent)
         my_prop;
         my_prop2;
+        unsaveable_property;
     end
     properties(Access=private)
         my_prop_ = 'beee';
         my_prop2_ = 10;
+        unsaveable_property_ = 'abra_cadabra'
     end
     methods
         function obj=config_base_tester()
             % constructor
             obj=obj@config_base(mfilename('class'));
+            obj.mem_only_prop_list_ = {'unsaveable_property'};
         end
-        function val = get.my_prop(this)
-            val = get_or_restore_field(this,'my_prop');
+        function val = get.my_prop(obj)
+            val = get_or_restore_field(obj,'my_prop');
             %val = this.my_prop;
         end
-        function this = set.my_prop(this,val)
-            config_store.instance().store_config(this,'my_prop',val);
+        function obj = set.my_prop(obj,val)
+            config_store.instance().store_config(obj,'my_prop',val);
         end
-        function val = get.my_prop2(this)
-            val = get_or_restore_field(this,'my_prop2');
+        function val = get.my_prop2(obj)
+            val = get_or_restore_field(obj,'my_prop2');
             %val = this.my_prop;
         end
-        function this = set.my_prop2(this,val)
-            config_store.instance().store_config(this,'my_prop2',val);
+        function obj = set.my_prop2(obj,val)
+            config_store.instance().store_config(obj,'my_prop2',val);
         end
         %------------------------------------------------------------------
         % genetic test methods
-              
+
         %------------------------------------------------------------------
         % ABSTACT INTERFACE DEFINED
         %------------------------------------------------------------------
-    
-        function fields = get_storage_field_names(class_instance)
+        function fields = get_storage_field_names(~)
             % helper function returns the list of the name of the structure,
             % get_data_to_store returns
             fields = {'my_prop','my_prop2'};
         end
-        function value = get_internal_field(this,field_name)
+        function value = get_default_value(this,field_name)
```

Comment:
Because it returns default value. The description of config_base class explains what is default value and what is value and if it is insufficient, I am ready to add more. But ages ago we closed this question.